docassemble/ALKilnTests/data/sources/languages.feature

```diff
@@ -0,0 +1,52 @@
+@langs
+Feature: Multi-language tests
+
+# Usually only run default test and skip or shorten others
+# When commanded, run all non-default language tests
+
+# We'll need to test these manually each time, as our users would,
+# and check that the automatic tests didn't trigger the language
+# tests
+
+# Not sure how to test urls like this: http://es.<other stuff>interview.yml
+# That's how wikipedia does it.
+
+# Tag use examples:
+# # No tags for default language only
+# @al_language # for all languages
+# @al_language @es # to isolate one language
```

Comment:
I think this would be confusing behavior; From my reading of things, `@es` wouldn't work (since `and not @al_language` would be added), and IMO it should just work.